### PR TITLE
Editor: Slides/Components, confirm before delete

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

File: client/components/Editor/ConfirmableDeleteButton.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+class ConfirmableDeleteButton extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            open: false
+        };
+    }
+
+    render() {
+        const { onConfirm, what } = this.props;
```
Comment (ktbee) [inline client/components/Editor/ConfirmableDeleteButton.jsx]: Heh i like the name `what`, but I find myself still asking "what is in this variable?". Perhaps we can make this more descriptive? Something like `itemType`?

**Comment by ktbee** [comment]
@rwaldron Let me know what you think of my new modal text!

**Comment by rwaldron** [comment]
@ktbee 
> Let me know what you think of my new modal text!

Perfect—mine was a bit wordy 😆

File: client/components/Editor/ConfirmableDeleteButton.jsx (hunk 1 above)
Comment (rwaldron) [reply client/components/Editor/ConfirmableDeleteButton.jsx]: I'm not opposed to that change, can you add another commit?
